File: fabric/src/main/java/com/ssblur/sauced/SaucedFabric.java
package com.ssblur.sauced;

import net.fabricmc.api.ClientModInitializer;
import net.fabricmc.api.ModInitializer;

public class SaucedFabric implements ModInitializer, ClientModInitializer {
    @Override
    public void onInitializeClient() {
        Sauced.clientInit();
    }

    @Override
    public void onInitialize() {
        Sauced.init();
    }
}

File: src/main/kotlin/com/ssblur/sauced/SaucedFabric.kt
package com.ssblur.sauced

import net.fabricmc.api.ClientModInitializer
import net.fabricmc.api.ModInitializer


object SaucedFabric : ModInitializer, ClientModInitializer {
    override fun onInitialize() {
        Sauced.init()
    }

    override fun onInitializeClient() {
        Sauced.clientInit()
    }
}